frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -20,25 +20,22 @@ import type { IInternalLayout } from '@altinn/ux-editor/types/global';
 import { PdfConfig } from '@altinn/ux-editor/components/Properties/PageConfigPanel/PdfConfig';
 
 export const PageConfigPanel = () => {
-  const { selectedFormLayoutName } = useAppContext();
+  const { selectedItem } = useAppContext();
```

Comment:
```suggestion
  const { selectedItem: page } = useAppContext();
```
Not necessary, but do you think it makes it more readable to name it as a page or layout in this context?

Reply:
Although the `PageConfigPanel` component currently deals with `page`, renaming selectedItem to `page` might exclude other types (`component` or `group`) defined in the `SelectedItem` type. In the future, if we need to add logic for components or layouts in the same component, we would have to revert the name change.I think Keeping `selectedItem` accurately reflects its purpose as the currently selected entity, regardless of type. What do you think?

Reply:
I agree :) Keep it as it is